app/src/main/contrib/components/Responsibilities/Expectations/ExpectationsDescription.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from "react";
-import {ExpectationMapping, NumberRange} from "../../../../shared/models/Generated";
+import {ExpectationMapping, NumberRange, Scenario} from "../../../../shared/models/Generated";
 import {CountriesList} from "./CountriesList";
 import {FileDownloadButton} from "../../../../shared/components/FileDownloadLink";
 
 interface ExpectationsDescriptionProps {
     expectationMapping: ExpectationMapping;
     groupId: String;
     touchstoneVersionId: String;
+    allScenarios: Scenario[];
 }
 
 export class ExpectationsDescription extends React.PureComponent<ExpectationsDescriptionProps> {
 
+    scenarioMapping(id: string) {
+        return <li key={id}>{id} : {this.props.allScenarios.find(s => s.id == id).description}</li>
```

Comment:
Seems weird to have a space before this colon. Otherwise this looks good.

Reply:
Fixed :+1:  will merge.